doc/manual/main_frontend_tools.rst

```diff
@@ -59,17 +61,28 @@ These are the top-level commands used to run and manage ARTIQ experiments. Not a
 -----------------------------------
 
 .. automodule:: artiq.frontend.artiq_session
+
+ARTIQ session manager. Automatically runs the master, dashboard and local controller manager on the current machine. The latter requires the ``artiq-comtools`` package to be installed. Requires access to device database, for the sake of the master.
```

Comment:
I would delete the last sentence, it is already documented under "artiq_master" and the error thrown if it is not supplied makes it immediately obvious.